tests/github-glue.test.ts

```diff
@@ -75,43 +75,58 @@ test("pull requests", async () => {
         await github.authenticate(owner);
         const content = Buffer.from("test data").toString("base64");
 
-        const branch = "test-branch";
-        const branchRef = `refs/heads/${branch}`;
-        const title = "Test pulls integration";
+        const branchBase = `ggg-test-branch-${process.platform}`;
+        const titleBase = `ggg Test pulls integration-${process.platform}`;
```

Comment:
Maybe we could also include the `id` of the run? That would have two benefits:

- two concurrent runs can't clash anymore
- it makes it easier to see *which* run left branches behind that then needed to be cleaned up later.

Reply:
Branches/PRs on Github and Azure will have names/titles like:
```
ggg-test-branch-win32_2020-12-18T23_56_40_422Z
```
While it does not ensure there won't be a conflict the risk is pretty low.  Branches/PRs should only be left behind if the run failed.  Will the raw log from failing runs provide the precision to determine the branch/PR?

Guess I am asking if this can be deferred until we see there is a problem?  There would be different vars to use based on GitHub workflow vs Azure pipeline (or the value could be set in the yaml).

Reply:
Okay, let's take it as-is, then.